src/components/WebCLI/WebCLI.tsx

```diff
@@ -56,16 +57,13 @@ const WebCLI = ({
   const [historyPosition, setHistoryPosition] = useState(0);
   const [inlineErrors, setInlineError, hasInlineError] = useInlineErrors();
   const inputRef = useRef<HTMLInputElement>(null);
-  const [output, setOutput] = useState<string[]>([]);
+  const [output, setOutput] = useState<HistoryItem[]>([]);
```

Comment:
This is currently stored for a single model, but a followup will introduce a multi model state.